README.md

```diff
@@ -15,18 +16,91 @@ Let us know how to improve this library. We'll be more than happy if you report
   - [setup & usage](#webhooks---setup--usage)
   - [event types](#webhooks---event-types) 
 - [Missing a feature?](#missing-a-feature)
-  
-## Prerequisities
 
-You need to [create an account](https://app.amio.io/signup) before you can use this library.
+## Prerequisities
+[Signup to Amio](https://app.amio.io/signup) and create a channel before using this library.
 
 ## Installation
 
 ```bash
 npm install amio-sdk-js --save
 ```
 
-## API 
+## Examples
```

Comment:
Ty **Examples** jsou super. Ještě bych tam jako alternativu napsal třeba **Quickstart**. 

Přijdou mi ty příklady ale strašně dlouhý. Použil bych v nich pouze jen **async** nebo jen **promise**. Chceš, aby je copy pastli a jeli, ne? Detailněji to rozpytváme v další sekci. Vynechal jsem i error handling. O tom se dočtou dále. Co takhle?

Nechal ses inspirovat Axiosem, ale Axios je HttpClient, proto na začátku show casujou error handling a finally. To mi nemusíme. Myslím, že máme jen ukázat, jak simple to s námi je.

### Send message
```
const AmioApi = require('amio-sdk-js').AmioApi

const amioApi = new AmioApi({
  accessToken: 'get access token at https://app.amio.io/administration/settings/api'
})

(async function() {
    await amioApi.messages.send({
      channel: {id: 'get CHANNEL_ID at https://app.amio.io/administration/channels/'},
      contact: {id: 'get CONTACT_ID at https://app.amio.io/administration/channels/{CHANNEL_ID}/contacts'},
      content: {
        type: 'text',
        payload: 'You rock!'
      }
    })
})()
```

### Receive message
```
const express = require('express')
const router = express.Router()
const WebhookRouter = require('amio-sdk-js').WebhookRouter

const amioWebhookRouter = new WebhookRouter({
  secrets: {
    // !!! CHANNEL_ID must be a string. Numbers can be converted to a different value !!!
    '{CHANNEL_ID}':'{SECRET}'
  }
})

amioWebhookRouter.onMessageReceived(function(data) {
  console.log('new message received from contact ' + data.contact.id + 'with content ' + data.content)
})

router.post('/webhooks/amio', function (req, res) {
  amioWebhookRouter.handleEvent(req, res)
})

```